.github/workflows/build-on-windows.yml

```
@@ -0,0 +1,34 @@
+name: Test build on Windows
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/build-on-windows.yml'
+      - RNReanimated.podspec
+      - android/build.gradle
+      - package.json
+  push:
+    branches:
+      - main
+
+env:
+  REACT_NATIVE_VERSION: "0.70.5"
+
+jobs:
+  build:
+    runs-on: windows-latest
+    concurrency:
+      group: build-on-windows-${{ github.ref }}
+      cancel-in-progress: true
+    steps:
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: '11'
+      - name: Create React Native app
+        run: npx react-native init app --version ${{ env.REACT_NATIVE_VERSION }}
+      - name: Install Reanimated
```

We can specify that we install the package from main branch.
```suggestion
      - name: Install Reanimated from main
```